### PR TITLE
Adding folder permission checks for the add folder and upload buttons

### PR DESCRIPTION
**Pull request by timkung**

### CODE REVIEW

File: javascript/src/sections/gallery/controller.js [hunk 1]
```diff
@@ -220,6 +222,8 @@ export class GalleryContainer extends SilverStripeComponent {
     const securityID = $(':input[name=SecurityID]').val();
 
     // TODO Make "add folder" and "upload" buttons conditional on permissions
```
Comment (scott1702) [inline javascript/src/sections/gallery/controller.js]: This TODO can be removed now

**Comment by timkung** [comment]
@flashbackzoo Just noting I think the tests are now breaking due to differences of versions for the `babelify` module and there are issues with the babel.presets options defined in the `framework/package.json`

**Comment by flashbackzoo** [comment]
Thanks. I'll take a look and make a PR against your fork if I can figure it out :smile:

**Comment by scott1702** [comment]
Any luck with that @flashbackzoo? Would be cool to get this merged in.

**Comment by flashbackzoo** [comment]
Sorry - haven't had a chance to look yet.

**Comment by chillu** [comment]
Rebased and merged as e22e4e42a59703076b47012586dc31260b551c0d

**Pull request closed without merging**